adapters/tlsio_mbedtls.c

```diff
@@ -732,7 +732,7 @@ int tlsio_mbedtls_send(CONCRETE_IO_HANDLE tls_io, const void *buffer, size_t siz
         }
         else
         {
-            tls_io_instance->send_complete_info.on_send_complete = on_send_complete;
+            tls_io_instance->send_complete_info.on_send_complete = on_send_complete_obj;
```

Comment:
gotta change this guy too

Reply:
Thanks @danewalton ! Any other changes you recommend?